doc/source/faq.rst

```diff
@@ -147,3 +148,43 @@ the ``ssh_password`` or ``ssh_port`` environment settings.
             }
         }
     }
+
+
+How do I dynamically generate the worker list?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In a small cluster it's sufficient to specify the list of workers in ``config.json``. 
+However, if you have a large or complex environment where workers are numerous 
+or short-lived, ``streamparse`` supports querying the nimbus server for a list of hosts.
+
+To trigger the dynamic lookup, the ``workers`` list should be empty or undefined.
+Disable this lookup (and use the empty list) by setting ``allow_worker_lookup: false``.
```

Comment:
I'm not aware of one, so I've changed the code such that only undefined lists will trigger a lookup.